modules/core/src/convert.hpp

```diff
@@ -11,7 +11,7 @@
 namespace cv
 {
 
-#if CV_SIMD
+#if (CV_SIMD || CV_SIMD_SCALABLE)
```

Comment:
I've fixed the compilation, but this merge still can not be built because of incompatible code being added in 5.x (uses operators with intrinsics, for example `vx_load_as` in this `#if` block). It means either separate pass with refactoring tool is needed or manual adaptation of the code.